client/benches/tps/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "peers": 4,
```

Comment:
I'd add a comment, since these peers are `test_network` based entities. As my benchmarking shows the performance degrades significantly once the peers are in separate processes.

Reply:
Create another benchmark with separate processes instead of threads?

Reply:
Ideally yes. But I think having a thread-based benchmark is also valuable.